src/main/java/uk/gov/ons/ctp/response/action/endpoint/ActionPlanEndpoint.java

```diff
@@ -69,20 +105,23 @@
   public final ResponseEntity<ActionPlanDTO> createActionPlan(
       @RequestBody @Valid final ActionPlanPostRequestDTO request, final BindingResult bindingResult)
       throws CTPException, InvalidRequestException {
-    log.info("Create action plan - action plan {}", request);
+    log.info(
+        "Creating action plan, Name: {}, Selectors: {}", request.getName(), request.getSelectors());
     if (bindingResult.hasErrors()) {
       throw new InvalidRequestException("Binding errors for create action plan: ", bindingResult);
     }
 
+    // Check if action plan with same name already exists
```

Comment:
So theres complaints from the codebeat about the ABC in random points one being here. Would it be worth splitting it up into two separate methods i.e. one that returns the action plan and one that checks if an action plan name already exists?